lib/commands/general.js

```diff
@@ -76,6 +76,15 @@ commands.getWindowSize = function () {
   return this.bootstrap.sendAction('getDeviceSize');
 };
 
+// For W3C
+commands.getWindowRect = function () {
```

Comment:
is the endpoint present in NO_PROXY list?

Reply:
This probably does need to be added, or else it won't work in Chrome (unless Chromedriver handles this endpoint?).

Reply:
then there should be isWebContext verification?

Reply:
Window size should be the native window size, no?

Reply:
Yes, it should be window size on mobile devices so far.